### DISCUSSION TITLE
Regex for profanity FormBuilderValidators.matchNot() to check a string

### DISCUSSION THREAD

**Discussion by jeejeestudio** [flutter-form-builder-ecosystem/form_builder_validators#113] [Ideas]
Is there a way to use a list of words as kind of regex, so that for example a username or other text string may not contain a profanity word ?
One could make a list of words they do not wish to be used and that could then be checked with for example matchNot.
Or a whole new Validator like FormBuilderValidators.profanity()

**Comment by martijn00** [comment]
There is `ContainsElementValidator` but you would want a contains not validator if i understand correctly?

**Comment by martijn00** [comment]
Implemented https://github.com/flutter-form-builder-ecosystem/form_builder_validators/pull/122

**Comment by jeejeestudio** [comment]
Thanks!
It seems I did not activated notifications for this thread. Sorry for that.
And thanks for your kindly effort!
